src/lib/FlightTasks/tasks/Auto/FlightTaskAuto.hpp

```diff
@@ -87,7 +87,6 @@ class FlightTaskAuto : public FlightTask
 
 protected:
 	void _setDefaultConstraints() override;
-	float _getMaxCruiseSpeed() {return MPC_XY_CRUISE.get();} /**< getter for default cruise speed */
```

Comment:
@bkueng @dagar Has the param abstraction been audited for runtime performance when being used in tight control loops? The previous design with locally cached variables was a deliberate choice.

Reply:
@LorenzMeier in this point there is still dead code that can be removed.
you might want to refer your question to this point of the code:
https://github.com/PX4/Firmware/blob/1638a1cb45c18cd6f4af89bee58b8dd2b30f8c51/src/lib/FlightTasks/tasks/Auto/FlightTaskAuto.cpp#L409

I am more than agree that the param abstraction mechanism must be more real-time oriented, than "code simplicity" oriented

Reply:
@LorenzMeier it's ultimately the same access pattern.
https://github.com/PX4/Firmware/blob/a9fc04b2f9ebc0cd48888c62a9c4fdd9beb77504/src/platforms/px4_param.h#L157